src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/RazorProjectEngine.cs

```diff
@@ -400,9 +400,7 @@ private static void AddDefaultFeatures(ImmutableArray<IRazorFeature>.Builder fea
             method.MethodName = "ExecuteAsync";
             method.ReturnType = $"global::{typeof(Task).FullName}";
 
-            method.Modifiers.Add("public");
-            method.Modifiers.Add("async");
-            method.Modifiers.Add("override");
+            method.UpdateModifiers("public", "async", "override");
```

Comment:
>method.UpdateModifiers("public", "async", "override"); [](http://example.com/codeflow?start=12&length=54)

I think this is right, but most of the other places that called the Update* methods were replacing code that did a Clear potentially followed by some Adds. I'm not seeing a Clear in the replaced code here, but I assume it was going to be empty anyways,.